CMakeLists.txt

```diff
@@ -146,17 +169,18 @@ if(JERRY_LIBC)
 endif()
 
 # Turn off stack protector
+if (USING_GCC_OR_CLANG)
 jerry_add_compile_flags(-fno-stack-protector)
+endif()
 
-# Debug information
-jerry_add_compile_flags(-g)
```

Comment:
I didn't; --  I wasn't confident in this change.  :-) 

I understand that -g was put in as a default variable b/c of an earlier bug in the code (maybe in jerry-main?...my memory doesn't get better as I age :-) -- and I know that interim fixes have made setting -g as the default unnecessary.

I like the flow that has the user set the cmake variables CMAKE_<LANG>_<DEBUG|RELEASE> to include/exclude settings -- this, I think, matches user expectations...

...but, as with all of these, I'll gladly make an argument and start a discussion, but I've got no ego in this: if I'm wrong here, I'll happily fix it (if we put -g back as an always-set flag, does it go back here, or, if not, where?)...